.well-known/apple-app-site-association

```diff
@@ -1,11 +1,22 @@
 {
```

Comment:
without `.nojekyll` you can't access `ketl.xyz/.well-known/`, it removes folder and files with `.`, `_` and other stuff